app/modules/appusage/scripts/install.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+MODULE_NAME="appusage"
+MODULE_CACHE_FILE="appusage.txt"
+
+CTL="${BASEURL}index.php?/module/${MODULE_NAME}/"
+
+# Get the scripts in the proper directories
+"${CURL[@]}" "${CTL}get_script/appusage" -o "${MUNKIPATH}preflight.d/appusage"
+
+if [ ! -f '/Library/Application Support/crankd/ApplicationUsage.py' ] || [ ! -f '/Library/Application Support/crankd/ApplicationUsage.py' ] ; then
+"${CURL[@]}" "${CTL}get_script/crankd_payload.zip" -o "/tmp/crankd_payload.zip"
```

Comment:
The zip file is missing. I wonder if it would be better to handle this as an external dependency like the location module.